department-of-veterans-affairs/next-build
# PR: Maintenance Banner #648

# Description

- Populate the maintenance banner in the header like how prod does.

## Ticket
Relates to #[648](https://github.com/department-of-veterans-affairs/next-build/issues/648). (or closes?)

## Developer Task

```[tasklist]
```

## Testing Steps

- Use the booklet to see if the the components are detected
- Use Inspect Chrome Developer Tools to match the wrapping at close as possible

## QA steps
What needs to be checked to prove this works?  

- Check that the booklet detects the va-maintenance-banner component

What needs to be checked to prove it didn't break any related things?  
N/A
What variations of circumstances (users, actions, values) need to be checked?
N/A

## Screenshots
Before:
![Maintenance banner](https://github.com/user-attachments/assets/e40e16ed-1c4a-42a1-b6e8-834863ff98bf)

After:
![Screenshot 2024-08-28 at 3 04 27 PM](https://github.com/user-attachments/assets/7b0ae13c-3416-44fb-a005-ae9dede445ae)



## Is this PR blocked by another PR? N/A

# Reviewer
<details>
<summary>Expand</summary>
### Reviewing a PR

This section lists items that need to be checked or updated when making changes to this repository. 

## Standard Checks

```[tasklist]
```

## Merging an Approved Layout

When merging a layout, you must ensure that the content type has been turned on for `next-build` inside the `CMS`. This CMS flag must be turned on for editors to preview their work using the next build preview server.

Resource types (layouts) that have not been approved by design should NOT be pushed to production. Ensure that [slug.tsx](../src/pages/[[...slug]].tsx) does not include your resource type if it is not approved.

The status of layouts should be kept up to date inside [templates.md](./templates.md). This includes QA progress, development progress, etc. A link should be provided for where testing can occur.

## Merging a Non-Approved Layout

Your new resource type should not be included inside [slug.tsx](../src/pages/[[...slug]].tsx). Items added here will go into production once merged into the `main` branch. It is imperative that we do not push anything live that has not been approved.

Ensure that this layout has been added to the [templates.md](./templates.md) file with the current status of the work.
</details>
